docs/bin/plugin_formatter.py

```diff
@@ -447,7 +452,11 @@ def main():
 
     env, template, outputname = jinja2_environment(options.template_dir, options.type)
 
-    mod_info, categories, aliases = list_modules(options.module_dir)
+    # Convert passed-in limit_to_modules to None or list of modules.
+    if options.limit_to_modules is not None:
+        options.limit_to_modules = [s.lower() for s in options.limit_to_modules.split(",")]
```

Comment:
I wouldn't block on this but stylewise, I do agree with alikins that it's better to do all modifications of things in options inside of the argument parsing functions and then set other variables if the code is outside of it.   (In my own code I tend to have a parse_options function that would combine generate_parser, validate_options, and also do transformations on the arguments like this.  That function would return the options data structure at the end and then I'd try to treat it as immutable thereafter).